services/requester/requester.go

```diff
@@ -159,12 +162,36 @@ func (e *EVM) SendRawTransaction(ctx context.Context, data []byte) (common.Hash,
 		return common.Hash{}, err
 	}
 
-	err := models.ValidateTransaction(tx)
-	if err != nil {
+	if err := models.ValidateTransaction(tx); err != nil {
 		return common.Hash{}, err
 	}
 
-	// todo do further validation
+	head := &types.Header{
+		Number:   big.NewInt(20_182_324),
+		Time:     uint64(time.Now().Unix()),
+		GasLimit: 30_000_000,
+	}
+	chainConfig := emulator.DefaultChainConfig
+	chainConfig.ChainID = e.config.EVMNetworkID
```

Comment:
maybe just rewrite this to this way 
 `emulator.NewConfig(WithChainID(chainID))`

Reply:
Good idea :ok_hand: 
Updated in https://github.com/onflow/flow-evm-gateway/pull/323/commits/73261f7e8c67bebc8238b1f4f0d0419fb56b1417